config/examples/Creality/CR-10 V3/Configuration_adv.h

```diff
@@ -1459,7 +1459,7 @@
    * This feature must be enabled with "M540 S1" or from the LCD menu.
    * To have any effect, endstops must be enabled during SD printing.
    */
-  //#define SD_ABORT_ON_ENDSTOP_HIT
+  #define SD_ABORT_ON_ENDSTOP_HIT
```

Comment:
If a endstop is hit during print that's not normal and me be safer to stop then, to avoid mechanical damage of the machine

Reply:
enabling this option could result in false positive readings due to vibrations or for example a pet touching the end stop that will result in failed prints.

Reply:
Hopefully I have no pet then !
But If my pet is able to go within the printer when it's running I still think it's safer to stop. 

And you still need to activate it in the LCD menu. on/off as you want.

Reply:
a number of these settings changes are of a personal preference nature and should remain on your fork.